client/landing/stepper/declarative-flow/internals/steps-repository/hundred-year-plan-step-wrapper/index.tsx

```diff
@@ -0,0 +1,254 @@
+
+	margin-top: 24px;
+`;
+
+const LearnMore = styled( Button )`
+	&.components-button.is-link {
+		color: var( --studio-gray-0 );
+		text-align: right;
+		font-size: 14px;
+		font-style: normal;
+		font-weight: 500;
+		line-height: 20px; /* 142.857% */
+		letter-spacing: -0.16px;
+		margin-top: 16px;
+
+		.gridicon {
+			margin-inline-start: 4px;
+		}
+	}
+`;
+
+const StyledFoldableCard = styled( FoldableCard )`
+	background: var( --studio-gray-100 );
+	color: var( --studio-gray-0 );
+	width: 100%;
+	&.foldable-card.card.is-expanded {
+		margin: 0;
+	}
+	&.foldable-card.is-expanded .foldable-card__content {
+		border: none;
+	}
+
+	&.card.foldable-card {
+		.foldable-card__header {
+			.foldable-card__main {
+				justify-content: flex-end;
+			}
+			.gridicons-chevron-down {
+				fill: var( --studio-gray-0 );
+			}
+			.foldable-card__action.foldable-card__expand {
+				.screen-reader-text {
+					right: 0;
+				}
+			}
+		}
+	}
+`;
+
+const WordPressLogoWrapper = styled.div`
+	position: absolute;
+	top: 24px;
+	left: 24px;
+`;
+
+function InfoColumnWrapper( { isMobile, children }: PropsWithChildren< { isMobile: boolean } > ) {
+	const planTitle = getPlan( PLAN_100_YEARS )?.getTitle();
+
+	return isMobile ? (
+		<StyledFoldableCard smooth hideSummary={ true } header={ planTitle }>
+			{ children }
+		</StyledFoldableCard>
+	) : (
+		<>{ children }</>
+	);
+}
+
+function InfoColumn( { isMobile, openModal }: { isMobile: boolean; openModal: () => void } ) {
+	const translate = useTranslate();
+
+	const productPrice = useSelect(
+		( select ) =>
+			select( ProductsList.store ).getProductBySlug( PLAN_100_YEARS )?.combined_cost_display,
+		[]
+	);
+
+	const planTitle = getPlan( PLAN_100_YEARS )?.getTitle();
+
+	return (
+		<>
+			<InfoColumnPlaceholder isMobile={ isMobile } />
+
+			<InfoColumnContainer
+				className="hundred-year-plan-step-wrapper__info-column-container"
+				isMobile={ isMobile }
+			>
+				<WordPressLogoWrapper>
+					<WordPressLogo size={ 24 } />
+				</WordPressLogoWrapper>
+
+				<HundredYearPlanLogo />
+				<Info isMobile={ isMobile }>
+					<Title>{ planTitle }</Title>
+					<Description isMobile={ isMobile }>
+						{ translate(
+							'Your stories, achievements, and memories{{br}}{{/br}}preserved for generations to come.',
+							{
+								components: {
+									br: <br />,
+								},
+							}
+						) }
+						<br />
+						{ translate( 'One payment. One hundred years of legacy.' ) }
+					</Description>
+					<LearnMore variant="link" onClick={ openModal }>
+						<>
+							{ translate( 'Learn More' ) }
+							<Gridicon icon="info-outline" size={ 16 } />
+						</>
+					</LearnMore>
+					<Price className={ ! productPrice ? 'is-price-loading' : '' }>{ productPrice }</Price>
+				</Info>
+			</InfoColumnContainer>
+		</>
+	);
+}
+
+function HundredYearPlanStepWrapper( props: Props ) {
+	const { stepContent, stepName, flowName, formattedHeader } = props;
+
+	const isMobile = useBreakpoint( '<960px' );
```

Comment:
`Non-blocking / Nitpick:` Should we eventually create a shared constant for the 960px mobile breakpoint? I see it used here and in the css of the styled components Ex. https://github.com/Automattic/wp-calypso/pull/81643/files#diff-3124b416cbc2da40eec0f5a7bc309cf67919183bcdc0dd80f43785cb9022b6b7R30

Reply:
Great suggestion! Done via https://github.com/Automattic/wp-calypso/pull/81643/commits/7d4e6476948a4f13c9c328546e3faa19facd80e4